application/views/userManagement/partial/editpermissions.php

```diff
@@ -6,10 +6,10 @@
 ?>
 
 <?=TbHtml::formTb(null, App()->createUrl('userManagement/saveUserPermissions'), 'post', ["id"=>"UserManagement--modalform"])?>
-<div class="modal-body selector--edit-permissions-container">
+<div class="modal-body selector--edit-permissions-container table-responsive">
```

Comment:
table-responsice are awfull for long size tabme, make table unusuable. I send a video on small screen